pixeltable/catalog/table_version.py

```diff
@@ -802,7 +833,7 @@ def _validate_update_spec(
         return update_targets
 
     def propagate_update(
-            self, plan: Optional[exec.ExecNode], where_clause: Optional[sql.ClauseElement],
+            self, plan: Optional['exec.ExecNode'], where_clause: Optional[sql.ColumnElement[bool]],
```

Comment:
Does it require 'bool'? If not, let's leave it off, we can't statically type-check exprs anyway.